basic/43-Arweave/Dev.md

```diff
@@ -0,0 +1,65 @@
+## Dev preparation
```

Comment:
Could you add a description to illustrate what's the purpose of this doc? What the reader will get after read this doc?

Reply:
basic information for preparing the developing